tests/puppeteer/campaigns/utils/globals.js

```diff
@@ -22,7 +22,7 @@ global.INSTALL = {
 global.BROWSER_CONFIG = {
   headless: JSON.parse(process.env.HEADLESS || true),
   timeout: 0,
-  slowMo: 25,
+  slowMo: parseInt(process.env.SLOWMO) || 5,
```

Comment:
```suggestion
  slowMo: parseInt(process.env.SLOWMO, 10) || 5,
```

Reply:
Is there a reason to specify the base when it's 10 by default ?

Reply:
Adding Radix param is recommended by lint